charts/infra-apps/examples/thanos.yaml

```diff
@@ -0,0 +1,12 @@
+thanos:
+  enabled: true
+  values:
+    existingObjstoreSecret: mysecret
+    storeGateway:
+      enabled: true
```

Comment:
What other typical things would we need here? Thanos needs an external blog store doesn't it?

Reply:
That is configured inside the `existingObjstoreSecret` 

But it does not need it to work.